audit/projects/k8s-staging-service-apis/buckets/artifacts.k8s-staging-service-apis.appspot.com/iam.json

```diff
@@ -10,21 +10,21 @@
     },
     {
       "members": [
-        "group:k8s-infra-staging-service-apis@kubernetes.io",
+        "deleted:group:k8s-infra-staging-service-apis@kubernetes.io?uid=515255850677684679501",
```

Comment:
Expected, this was the groups part of https://github.com/kubernetes/k8s.io/pull/1954 deploying